docs/src/main/sphinx/connector/sqlserver.rst

```diff
@@ -84,6 +84,23 @@ catalog named ``sales`` using the configured connector.
 
 .. include:: jdbc-common-configurations.fragment
 
+Specific configuration properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The SQL Server connector supports additional catalog properties to configure the
+behavior of the connector and the issues queries to the database.
+
+.. list-table::
+  :widths: 30, 70
+  :header-rows: 1
+
+  * - Property name
+    - Description
+  * - ``sqlserver.snapshot-isolation.disabled``
```

Comment:
From a code perspective I think it would be better to have sqlserver.snapshot-isolation.enabled with default true .. but we cant change that with just a doc change ..

Reply:
@ebyhr ... I talked to @martint when we worked on the release notes and he agrees .. the property name and default value should be refactored ... can you file a ticket for that and slot that into the queue somehow?